app/services/dismiss_topics.rb

```diff
@@ -8,6 +8,7 @@ def initialize(user, topics_scope)
 
   def perform!
     DismissedTopicUser.insert_all(rows) if rows.present?
+    @rows.map { |row| row[:topic_id] }
```

Comment:
return dismissed topics ids. rows are memoized so it should not affect performance